intfconfd/Makefile.am

```diff
@@ -0,0 +1,15 @@
+INCLUDES = -I $(top_srcdir) -I $(top_srcdir)/cfgorch
+
+bin_PROGRAMS = intfconfd
+
+if DEBUG
+DBGFLAGS = -ggdb -DDEBUG
+else
+DBGFLAGS = -g
+endif
+
+intfconfd_SOURCES = intfconfd.cpp intfconf.cpp $(top_srcdir)/cfgorch/cfgorch.cpp
```

Comment:
>cfgorch [](start = 61, length = 7)

suggest merge "cfgorch", "intfconfd" into one directory

Reply:
Yes,  cfgagent might be a good name for the directory.  Will move intfconfd, vlanconfd and debug cli implmentation cfgmgrtest there.  Actually that was the initial arrangement.